datacite.yml

```diff
@@ -0,0 +1,75 @@
+# Metadata for DOI registration according to DataCite Metadata Schema 4.1.
+# For detailed schema description see https://doi.org/10.5438/0014
+
+authors:
+  - firstname: Michael
+    lastname: Hanke
+    id: "ORCID:0000-0001-6398-6370"
+
+  - firstname: Adina
+    lastname: Wagner
+    id: "ORCID:0000-0003-2917-3450"
+
+  - firstname: Falko
```

Comment:
Quick question: Were those authors determined on the basis of commits? If so, I actually think that @loj should be on here too, and @christian-monch should include himself also in all those files

Reply:
1. Yes, authors were determined, based on commits.
2. Yes, then Laura should be in there too.
3. Yes, if we take commit-based authorship, christian-monch should add himself.